client/my-sites/theme/controller.jsx

```diff
@@ -30,31 +30,33 @@ export function fetchThemeDetailsData( context, next ) {
 		return next();
 	}
 
+	const siteId = 'wpcom';
```

Comment:
Okay, will change. I guess I had future usage for logged-in users in mind, but that's a long shot right now.